CMakeLists.txt

```diff
@@ -416,20 +418,25 @@ if (USE_SDL_MIXER)
 endif ()
 
 # Link spdlog
-FetchContent_Declare(
-	spdlog
-	GIT_REPOSITORY https://github.com/gabime/spdlog.git
-	GIT_TAG v1.14.1
-)
+if (USE_SYSTEM_SPDLOG)
+	find_package(spdlog REQUIRED)
+else ()
+	FetchContent_Declare(
+		spdlog
+		GIT_REPOSITORY https://github.com/gabime/spdlog.git
+		GIT_TAG v1.14.1
+	)
 
-set(SPDLOG_BUILD_EXAMPLE OFF)
-set(SPDLOG_DISABLE_DEFAULT_LOGGER ON)
-set(SPDLOG_ENABLE_PCH ${USE_PCH})
-set(SPDLOG_USE_STD_FORMAT ON)
```

Comment:
We're using spdlog via FetchContent because we rely on that specific configuration, especially `SPDLOG_USE_STD_FORMAT`. As spdlog is linked statically into the result, I don't see why a build for nix would need to change that?

Reply:
The nix build does not have internet access, so `FetchContent` does not work. Using spdlog from nixpkgs and setting the apropriate configuration there was just the easiest way to get a working build.